src/tsolvers/lrasolver/LRASolver.cc

```diff
@@ -312,9 +312,9 @@ LRASolver::getInterpolant( const ipartitions_t & mask , map<PTRef, icolor_t> *la
 
 
 
-PTRef LRASolver::getDecomposedInterpolant(const ipartitions_t &mask, map<PTRef, icolor_t> *labels) {
+PTRef LRASolver::getDecomposedInterpolant(const ipartitions_t &mask, map<PTRef, icolor_t> *labels, PartitionManager &pmanager) {
     LRA_Interpolator interpolator{logic, explanation, explanationCoefficients, mask, labels};
     icolor_t color = config.getLRAInterpolationAlgorithm() == itp_lra_alg_decomposing_strong ? icolor_t::I_A : icolor_t::I_B;
```

Comment:
Can we use PartitionManager::part to avoid using global symbols?

Reply:
Sounds good, again, I would do it during LRA Interpolation refactoring.